bin/harvey

```diff
@@ -24,7 +24,7 @@ try {
 }
 
 try {
-	config = (options.configFile) ? loadJson(options.configFile) : {};
+	config = (options.configFile) ? require(path.resolve(options.configFile)) : {};
```

Comment:
It looks like this would change the way that the config loading is happening. Currently there is some caching involved with the loadJson function.